index.js

```diff
@@ -263,6 +279,16 @@ function cloneAndFilter() {
                 && !fd.name.endsWith('.rst')
                 && !fd.name.endsWith('.gif')
                 && !fd.name.endsWith('.webp')
+                && !fd.name.endsWith('.pkl')
+                && !fd.name.endsWith('.pickle')
+                && !fd.name.endsWith('.json')
+                && !fd.name.endsWith('.tar.gz')
+                && !fd.name.endsWith('.zip')
+                && !fd.name.endsWith('.tar')
+                && !fd.name.endsWith('.gz')
+                && !fd.name.endsWith('.so')
+                && !fd.name.endsWith('.dll')
+                && !fd.name.endsWith('.dylib')
```

Comment:
As an idea: 

Wouldn't it be better to retrieve the list of file extensions associated with a programming languages insted banning of specific files? 

For example, we could reference this curated list: https://gist.github.com/ppisarczyk/43962d06686722d26d176fad46879d41